### PR TITLE
fixed bug with wiki report submission

### PR DESCRIPTION
**Pull request by Yomzy2223**
# Wiki report submission bug

_Fixed bug with wiki report submission_

## How should this be tested?

1. Click on "Report to us" button from wiki details
2. Write in some texts then click "Confirm"

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2911

### CODE REVIEW

**Review by Adebesin-Cell — COMMENTED** [review]
Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.

**Comment by Yomzy2223** [comment]
> Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.

Resulted in parameter overload, as the backend only takes report and wikiId

**Comment by Adebesin-Cell** [comment]
> > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> 
> Resulted in parameter overload, as the backend only takes report and wikiId

Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.

**Comment by AshakaE** [comment]
> > > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> > 
> > 
> > Resulted in parameter overload, as the backend only takes report and wikiId
> 
> Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.

It's not needed, if you want to do a more robust feedback, you can use contentFeedback api

```gql
contentFeedback(
contentId: String
userId: String
site: ContentFeedbackSite!
message: String
rating: Int
input: String
output: String
reportType: String
): Boolean!
```

**Comment by Adebesin-Cell** [comment]
> > > > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> > > 
> > > 
> > > Resulted in parameter overload, as the backend only takes report and wikiId
> > 
> > 
> > Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.
> 
> It's not needed, if you want to do a more robust feedback, you can use contentFeedback api
> 
> ```graphql
> contentFeedback(
> contentId: String
> userId: String
> site: ContentFeedbackSite!
> message: String
> rating: Int
> input: String
> output: String
> reportType: String
> ): Boolean!
> ```

Perhaps we use this instead of having two separate endpoints that does same thing but one is a smaller version of the other.

**Comment by Yomzy2223** [comment]
> > > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> > 
> > 
> > Resulted in parameter overload, as the backend only takes report and wikiId
> 
> Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.

I have confirmed with @AshakaE, userId is not needed.

**Comment by Adebesin-Cell** [comment]
> > > > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> > > 
> > > 
> > > Resulted in parameter overload, as the backend only takes report and wikiId
> > 
> > 
> > Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.
> 
> I have confirmed with @AshakaE, userId is not needed.

Yep! Got it. Let’s use content feedback

**Comment by Yomzy2223** [comment]
> > > > > Why we remove userId? I think it's a necessary data to keep, we could make it optional if it's not meant to be recorded.
> > > > 
> > > > 
> > > > Resulted in parameter overload, as the backend only takes report and wikiId
> > > 
> > > 
> > > Perhaps confirm with @AshakaE about the user id, cos it seems to be passed as a required param.
> > 
> > 
> > I have confirmed with @AshakaE, userId is not needed.
> 
> Yep! Got it. Let’s use content feedback

Alright

File: public/locales/en/home.json
Comment (invisiblemask) [inline public/locales/en/home.json:1]: This is not related to this PR

**Review by invisiblemask — COMMENTED** [review]
There are so many unrelated changes on this PR, we can limit this PR to it's description

File: public/locales/en/home.json
Comment (Yomzy2223) [reply public/locales/en/home.json:1]: I pulled & merged changes from main branch. That's why. I'll just create a fresh pr now, for just the fix alone, so it's easier to review.

Comment (invisiblemask) [reply public/locales/en/home.json:1]: Oh okay,or you could revert other commits and leave yours?
